presentational/SearchBar.js

```diff
@@ -68,54 +38,88 @@ const StyledSearchIcon = styled(SearchIcon)`
   width: 20px;
 `
 
-export default ({ translationQuery, onTranslationQueryChange }) => (
+const LanguageWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin: 5px 0;
+  flex-direction: column;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+`
+
+const LanguageSelectWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+const LanguageSelect = ({ language, onClick }) => {
+  return (
+    <>
+      <Language>{language ? 'French' : 'English'}</Language>
+      <SwitchButton onClick={() => onClick()}>
```

Comment:
This would be the same

```suggestion
      <SwitchButton onClick={onClick}>
```